lib/blocks.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Functions related to editor blocks for the Gutenberg editor plugin.
+ *
+ * @package gutenberg
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'Silence is golden.' );
```

Comment:
This is kind of a paradox. :)

I usually do `defined( 'ABSPATH' ) || die;` to keep it short.